sdk/core/System.ClientModel/tests/client/ModelReaderWriter/Models/DiscriminatorSet/BaseModel.cs

```diff
@@ -12,6 +13,24 @@ public abstract class BaseModel : IJsonModel<BaseModel>
     {
         private Dictionary<string, BinaryData> _rawData;
 
+        public static implicit operator BinaryContent(BaseModel baseModel)
```

Comment:
Adding tests for `BinaryContent(model)` overloads that come in from the [original serialization PR](https://github.com/Azure/azure-sdk-for-net/pull/35946)